common/lib/xmodule/xmodule/js/src/html/edit.js

```diff
@@ -1233,25 +1237,25 @@
     }
 
     HTMLEditingDescriptor.prototype.setupTinyMCE = function(ed) {
-      ed.addButton('wrapAsCode', {
+      ed.ui.registry.addButton('wrapAsCode', {
 
         /*
         Translators: this is a toolbar button tooltip from the raw HTML editor displayed in the browser when a user needs to edit HTML
          */
-        title: gettext('Code block'),
-        image: baseUrl + "images/ico-tinymce-code.png",
-        onclick: function() {
+        tooltip: gettext('Code block'),
+        icon: 'code-sample',
```

Comment:
TinyMCE5 only allows svg icons and have different way to add new icon ([ref](https://www.tiny.cloud/docs/ui-components/typesoftoolbarbuttons/#configoptions)), overe here we are using a builtin icon.

Old png icon: `{ }`
New svg icon (code-sample): `{ ; }`

(Check screenshot in the PR description for UI)